loop/middlewares.js

```diff
@@ -101,6 +101,12 @@ function logMetrics(req, res, next) {
         line.participants = req.roomParticipantsCount;
       }
 
+      if (req.hasOwnProperty("roomStatusData")) {
+        Object.keys(req.roomStatusData).forEach(function(k) {
+          line[k] = req.roomStatusData[k];
```

Comment:
Yeah, we actually should do this rather than adding our information to the req object everywhere :)